jobserver/releases.py

```diff
@@ -236,6 +236,12 @@ def serve_file(request, rfile):
         # bypass DRFs renderer framework and just serve bytes
         response = FileResponse(path.open("rb"))
 
+        # append a charset to the existing content-type header, being careful
+        # just in case the existing value is empty
+        content_type = response.headers.get("Content-Type")
+        joiner = "; " if content_type else ""
```

Comment:
I think this is buggy - should probably only add a charset if the content-type starts with `text/`, right? We don't want images to be served with a charset?

Reply:
oooh good shout 👍